add gpon and tapi as subsystem to hotplug rules of procd

diff --git a/package/system/procd/files/hotplug.json b/package/system/procd/files/hotplug.json
--- a/package/system/procd/files/hotplug.json
+++ b/package/system/procd/files/hotplug.json
@@ -70,7 +70,7 @@
 	],
 	[ "if",
 		[ "eq", "SUBSYSTEM",
-			[ "net", "input", "usb", "usbmisc", "ieee1394", "block", "atm", "zaptel", "tty", "button" ]
+			[ "net", "input", "usb", "usbmisc", "ieee1394", "block", "atm", "zaptel", "tty", "button", "gpon", "tapi" ]
 		],
 		[ "exec", "/sbin/hotplug-call", "%SUBSYSTEM%" ]
 	],
